BeVolt/BSP/Simulator/DataGeneration/CAN.py

```diff
@@ -0,0 +1,13 @@
+import csv
+
+#path name to file
+file = "Data/CAN.csv"
+
+"""
+@brief send a CAN message as a specific system
+    Function called by simulate.py
+"""
+def Send_Message(id, message[8], length):

```

Comment:
Make sure that you add the proper functionality to the simulate.py engine so that you can actually test this. This code won't run properly due to a syntax error here. In python, ```message[8]``` isn't a valid parameter. Everything is dynamically typed in Python, so the parameter would just be ```message``` and it would be understood that that is supposed to be a list. 

Also, I think we ended up deciding that you actually didn't need to put the row labels in the file since it's a CSV and not viewed by us anyway.